src/containers/mentor-home-page/become-a-tutor/BecomeATutor.js

```diff
@@ -1,16 +1,44 @@
 import { Box } from '@mui/material'
+import FirstStep from '~/components/become-tutor-steps/firstStep/FirstStep'
 import StepWrapper from '~/components/step-wrapper/StepWrapper'
+import useForm from '~/hooks/use-form'
+import { firstName, lastName, country, city } from '~/constants/validation/become-tutor'
 
-const stepLabels = ['General info', 'Languages', 'Study category', 'Experience', 'Documents', 'Photo and Video']
+const BecomeATutor = () => {
+  const stepLabels = ['General info', 'Languages', 'Study category', 'Experience', 'Documents', 'Photo and Video']
 
-const childrenArr = [
-  <Box key="1">1</Box>,
-  <Box key="2">2</Box>
-]
+  const { handleSubmit, handleChange, handleBlur, data, errors } = useForm({
+    initialValues: {
```

Comment:
You can move initial values out of the component.

Reply:
done